graphql/pod_resolver.go

```diff
@@ -10,13 +10,29 @@ import (
 	"github.com/evergreen-ci/evergreen/model/event"
 	"github.com/evergreen-ci/evergreen/model/task"
 	"github.com/evergreen-ci/evergreen/rest/model"
-	restModel "github.com/evergreen-ci/evergreen/rest/model"
 	"github.com/evergreen-ci/utility"
 )
 
-// Type is the resolver for the type field.
-func (r *podResolver) Type(ctx context.Context, obj *model.APIPod) (string, error) {
-	return string(obj.Type), nil
+// Events is the resolver for the events field.
+func (r *podResolver) Events(ctx context.Context, obj *model.APIPod, limit *int, page *int) (*PodEvents, error) {
```

Comment:
Was this meant to be part of this PR?

Reply:
This is a reorganization of the file that was needed to make `make gqlgen` succeed, which was necessary as part of this PR.